README.md

```diff
@@ -5,11 +5,19 @@
 csv2wiki requires Python 3.  So far, only
 [MediaWiki](https://mediawiki.org/), version 1.28.2 or later, is supported,
 but it would not be hard to extend it to support other wiki APIs.  Basic
-usage:
+usage from current directory:
 
-        $ csv2wiki -c CONFIG [OPTIONS] CSV
+        $ python3 -m csv2wiki -c CONFIG [OPTIONS] CSV
 
-Run `python3 ./csv2wiki --help` to see complete usage.  Summary:
+or after installing as a package
+
+        $ pip install -e .
```

Comment:
So this tells me how to install it as a package, but doesn't tell me how to run it after installing.  Would be worth making this more explicit I think.

Reply:
Ah, I see where I was confused.  I was looking for something like this:
```
or after installing as a package, to run from any directory:

  $ pip install -e .
  $ python3 -m csv2wiki -c CONFIG [OPTIONS] CSV
```
Here's another way to structure it:
```
Install as a package in order to run in any directory (using command above):
  
  $ pip install -e .
```